src/one_dragon/devtools/python_launcher.py

```
@@ -289,10 +289,14 @@ def fetch_latest_code(ctx: OneDragonEnvContext) -> None:
         return
     _configure_runtime_logger()
+    from one_dragon.envs.git_service import GitSyncStatus
+
     progress_callback = create_git_progress_callback()
-    success, msg = ctx.git_service.fetch_latest_code(progress_callback=progress_callback)
-    if success:
+    status, msg = ctx.git_service.fetch_latest_code(progress_callback=progress_callback)
+    if status is GitSyncStatus.SUCCESS:
         print_message(gt('代码更新完成'), "PASS")
+    elif status is GitSyncStatus.RUNTIME_INCOMPATIBLE:
+        print_message(f"{msg}，请更新集成启动器", "ERROR")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)